packages/row/.storybook/config.js

```diff
@@ -1,9 +1,16 @@
-/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import React from 'react'
+import addons from '@storybook/addons'
+import { addDecorator, configure } from '@storybook/react'
 
-import { configure } from '@storybook/react';
+import centerDecorator from '@pluralsight/ps-design-system-storybook-addon-center'
+import themeDecorator from '@pluralsight/ps-design-system-storybook-addon-theme'
 
-function loadStories() {
-  require('../stories');
+addDecorator(fn => <div style={{ maxWidth: 'calc(100vw - 40px)' }}>{fn()}</div>)
+addDecorator(centerDecorator)
```

Comment:
I think it might be best not to use this decorator here.  It's interfering with the full-width nature of the row component.  I can't determine what styles are constraining it after a quick look, but this is what I see:

With center:

![screen shot 2018-12-11 at 9 49 16 am](https://user-images.githubusercontent.com/552044/49816035-48678e00-fd2a-11e8-8d50-d12ba7769e4a.png)

As before, without:

![screen shot 2018-12-11 at 9 48 17 am](https://user-images.githubusercontent.com/552044/49816048-4d2c4200-fd2a-11e8-9254-8eaec4a934d8.png)

It made me concerned that the full-width flexibility of the Row may have been broken (which it was not, as I discovered when I disabled this decorator).  Perhaps it's better to go without.

Reply:
good point - i'll do something different